src/components/BankTransactionList/BankTransactionList.tsx

```diff
@@ -27,27 +35,91 @@ export const BankTransactionList = ({
   showDescriptions,
   showReceiptUploads,
   showTooltips,
+  _showBulkSelection = false,
 }: BankTransactionListProps) => {
+  const { selectedIds } = useSelectedIds()
```

Comment:
nit (non-functional): I like that the hook is succinct, but I feel like defining this hook like `useBulkSelectionSelectedIds` might convey more information about which store it is related to, similar to how hooks are defined in `BankTransactionsRouteStoreProvider.tsx`.

```suggestion
  const { selectedIds } = useBulkSelectionSelectedIds()
```


I think this brings readability improvements but also import experience would improve: as a dev, when you know you need the bulk selection store, you would simply type `useBulkSelection` and all the options would show up.